compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/BIRGen.java

```diff
@@ -2317,7 +2317,7 @@ private BIROperand getQNameOP(BLangExpression qnameExpr, BIROperand keyRegIndex)
             return keyRegIndex;
         }
 
-        BIRVariableDcl tempQNameVarDcl = new BIRVariableDcl(qnameExpr.type,
+        BIRVariableDcl tempQNameVarDcl = new BIRVariableDcl(symTable.anyType,
```

Comment:
why this is anyType ?

Reply:
Couldn't find a corresponding type for `XMLQName`. It's implemented as a RefValue, so used anyType to mimic that.

Reply:
Ack. An interesting problem, the spec doesn't mention a type for this. I think it should be an XML attribute type. Need to verify with James.